<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;Form Preview&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="publishFormButton">
   <property name="geometry">
    <rect>
     <x>579</x>
     <y>673</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Publish</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Set Publish Date:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>530</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Set End Date:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="startDateTime">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>530</y>
     <width>194</width>
     <height>25</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>d MMM yyyy HH:mm</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="endDateTime">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>530</y>
     <width>194</width>
     <height>25</height>
    </rect>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectionMode::CorrectToNearestValue</enum>
   </property>
   <property name="displayFormat">
    <string>d MMM yyyy HH:mm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editFormButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Edit Form</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="formPreviewScroll">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>701</width>
     <height>451</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="questionsContainer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
